### PR TITLE
Add sqsIngester

### PR DESCRIPTION
**Pull request by djfritz**
@floren

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -25,6 +24,7 @@ require (
 	github.com/gravwell/filewatch/v3 v3.3.3
 	github.com/gravwell/ingest/v3 v3.3.7
 	github.com/gravwell/netflow/v3 v3.2.3
+	github.com/gravwell/timegrinder v3.2.2+incompatible
```
Comment (floren) [inline go.mod:27]: This shouldn't be in here... did you import github.com/gravwell/timegrinder at one point, build, then realize you needed /v3?

Comment (djfritz) [reply go.mod]: Odd - running `go mod tidy` fixed this. PTAL

**Comment by djfritz** [comment]
@floren PTAL - We also need to get @traetox involved re: go.mod, as he's made other changes to go.mod from:

https://github.com/gravwell/ingesters/commit/77804969fdfb8a162c9e0af8d24bb38d36c54141
https://github.com/gravwell/ingesters/commit/30bdb509c8504bdcb11d2833ffc27d284e36e0db

That look specific enough to need attention. @traetox ?

**Comment by traetox** [comment]
we just need to merge them.  The changes in go.mod refer to some specific commits on the winevent ingester and supporting libraries.  We want to make sure we are pointing at the right winvent stuff.

**Comment by floren** [comment]
Merged.

**Pull request merged**